pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/grpc/GrpcStatus.java

```diff
@@ -17,7 +17,9 @@
 package com.hedera.pbj.runtime.grpc;
 
 /**
- * Status codes for gRPC. These are added to a {@link GrpcException} when an error occurs.
+ * Status codes for gRPC. These are added to a {@link GrpcException} when an error occurs. The ordinal of the enum
+ * <strong>exactly matches</strong> the status code in the gRPC protocol. The order in which these enums are defined
+ * is critical.
```

Comment:
Awesome! Please see my comment for a previous revision - the individual comments could be removed then probably.